platform/nodejs/index.js

```diff
@@ -211,6 +256,7 @@ if ( typeof module !== 'undefined' && typeof exports !== 'undefined' ) {
 
 export {
     FilteringContext,
```

Comment:
I've left this here though it should be possible to remove `FilteringContext`, `pslInit`, and `useRawLists`.